app/templates/components/event-card.hbs

```diff
@@ -1,28 +1,39 @@
-<div class="ui fluid event card">
-  <a class="image" href="{{href-to 'public' event.identifier}}">
-    {{widgets/safe-image src=(if event.large event.large event.placeholderUrl)}}
-  </a>
-  <div class="content">
-    <a class="header" href="{{href-to 'public' event.identifier}}">
-      <span>{{event.name}}</span>
-    </a>
-    <div class="meta">
-      <span class="date">
-        {{moment-format event.startTime 'ddd, MMM DD HH:mm A'}}
-      </span>
+<div class="{{if isWide 'event wide ui grid row'}}">
+  {{#if isWide}}
+    <div class="ui card three wide computer six wide tablet six wide mobile column">
+      <a class="image" href="{{href-to 'public' event.identifier}}">
```

Comment:
Hide it only for the wideLayout ... for the landing page let the image be there even on mobile. 
@CosmicCoder96